### ISSUE TITLE
Update inputs design page to state icons used

### ISSUE THREAD

**Issue by StommePoes** [Pearson-Higher-Ed/design#232]
On the checkboxes and radio buttons sections under the Redlines header, it states for checkboxes:
>Unselected checkboxes use the checkbox-off-18 icon
Selected checkboxes use the checkbox-on-18 icon

However this is not true-- the design shows a light-coloured checkbox for "on" however the icon called checkbox-on-18 is black.

Beyond that, it is not possible to use either the radio or checkbox SVGs for any of them (for checkboxes, check-sm-18 is used; for radios, likely new-notification-9 will be used (for example, it's not possible to use the SVG for the disabled state where the background of the SVG is disabled-grey only between the outer border and the dot but not extend past the corners; background is background is background).

I think the docs should be updated to state checkboxes use "check-sm-18" and "new-notification-9" (however to make it match dot-size in the radio-18, it will be resized to 7px from 9px) so developers reading both the design spec and perusing the current Elements code see a match.

**Comment by parkerpearson** [comment]
@StommePoes so is this a change that needs to be made in the inputs, the icons, or both?

**Comment by StommePoes** [comment]
Inputs only

**Comment by parkerpearson** [comment]
So you're saying we don't really use icons at all in the implementation of radio buttons or checkboxes? Would it make more sense to just remove those from the icons component then?

cc/ @Van2Yang

**Comment by StommePoes** [comment]
correct. Jason uses the small check and I use the notification dot, if you want a hangout I can show you how and why...
